test/fixtures/ES6/arrow-function/complex-rest-in-arrow-not-allowed.failure.json

```diff
@@ -1 +1 @@
-{"index":6,"lineNumber":1,"column":7,"message":"Error: Line 1: Unexpected token [","description":"Unexpected token ["}
\ No newline at end of file
+{"index":10,"lineNumber":1,"column":11,"message":"Error: Line 1: Strict mode function may not have duplicate parameter names","description":"Strict mode function may not have duplicate parameter names"}
```

Comment:
Seems this test should be renamed to `complex-rest-in-arrow-with-duplicated-names-not-allowed.js` and a new test for complex rest should be created, showing it's allowed in arrows: `(a, ...[b]) => c`